src/python/CMSMonitoring/StompAMQ7.py

```diff
@@ -76,6 +76,8 @@ class StompyListener(stomp.ConnectionListener):
     def __init__(self, logger=None):
         if logger:
             self.logger = logger
+            # avoid flood of INFO logs from listener and transport
```

Comment:
@amaltaro  How about making this more flexible. Their logging level is not matching with us according to our usage. Their INFO logs are DEBUG for us, their WARNING is INFO for us. May be we can set stomp log level as one level up for any given logger level.

Let's say you want to debug everything in WMCore, but you'll always get WARNING level logs from Stomp for now.

So, my idea is like: 
```
givenLoggerLevel = logging.getLogger().level
logging.getLogger("stomp.py").setLevel(givenLoggerLevel + 10)
```
- if you give logger with INFO(20) level, stomp one will be WARNING(30).
- if you give logger with DEBUG(10) level, stomp one will be INFO(20).

What do you think?

Reply:
Sounds good. Let me change this code.